### ISSUE TITLE
Add README

### ISSUE THREAD

**Issue by GeckoEidechse** [R2Northstar/Atlas#1]
I know docs are in the works but we should at least already have a basic README that says what Atlas is, i.e. MasterServer for Northstar.

**Comment by pg9182** [comment]
Yep, I'm planning to have that done before the NS release tonight.
